lib/didResolver.js

```diff
@@ -0,0 +1,150 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import {didIo} from '@bedrock/did-io';
+import {DidResolutionResult} from './DidResolutionResult.js';
+import logger from './logger.js';
+
+/**
+ * Resolves a did after validating it.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.did - The did or didUrl being resolved.
+ * @param {object} options.didOptions - Options passed to the resolver.
```

Comment:
Can the description be more clear? To me the description for `didOptions` sounds the same as description for `resolverOptions`. How are the two different?

Reply:
This is an excellent point and I made changes here: https://github.com/digitalbazaar/bedrock-did-resolver-http/pull/3/commits/e83a8ce5cadc37d62f1dc8590dafc7db4a50f8bf